python/cudf/cudf_pandas_tests/test_cudf_pandas.py

```diff
@@ -2079,3 +2080,14 @@ def test_pickle_round_trip_proxy_numpy_array(array):
     np.testing.assert_equal(
         pickle.load(pickled_proxy_arr), pickle.load(pickled_arr)
     )
+
+
+def test_pandas_objects_not_callable():
+    series = xpd.Series([1, 2, 3])
+    dataframe = xpd.DataFrame({"a": [1, 2, 3], "b": [4, 5, 6]})
+    index = xpd.Index([1, 2, 3])
+    range_index = xpd.RangeIndex(start=0, stop=10, step=1)
+    assert isinstance(series, Callable)
+    assert isinstance(dataframe, Callable)
+    assert isinstance(index, Callable)
+    assert isinstance(range_index, Callable)
```

Comment:
Is this missing a `not`? Can we also verify that the _classes_ are callable (via their constructor), but instances are not? (Maybe the class check should be in another test.)
```suggestion
    assert not isinstance(series, Callable)
    assert not isinstance(dataframe, Callable)
    assert not isinstance(index, Callable)
    assert not isinstance(range_index, Callable)

    assert isinstance(xpd.Series, Callable)
    assert isinstance(xpd.DataFrame, Callable)
    assert isinstance(xpd.Index, Callable)
    assert isinstance(xpd.RangeIndex, Callable)
```

Reply:
Yup, my bad. I meant to have `not`